pkgs/data/themes/pop-gtk/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, sassc
+, gtk3
+, inkscape
+, optipng
+, gnome3
+, gtk-engine-murrine
+, gdk-pixbuf
+, librsvg
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pop-gtk-theme";
+  version = "8ddf0fcad13c6a3f4fab2772b45561d74a8b4cce";
```

Comment:
Usually we use the date of the commit as the version: 2019-11-13

Reply:
Noted, thank you